testdata/generate-remote-static/component.yaml

```diff
@@ -4,6 +4,6 @@ path: ./manifests
 subcomponents:
   - name: "keyvault-sub"
     source: https://raw.githubusercontent.com/Azure/kubernetes-keyvault-flexvol/master/deployment/kv-flexvol-installer.yaml
-    method: remote-url
+    method: http
     type: static
     path: "./tmp/keyvault-sub"
```

Comment:
For future reference, please setup your editor to ensure newlines on end of files (makes files more grep friendly). This is the `files.insertFinalNewline` option in vscode.